Swift/Tests/ReplicatorTest+PendingDocIds.swift

```diff
@@ -21,6 +21,9 @@ import XCTest
 import CouchbaseLiteSwift
 
 class ReplicatorTest_PendingDocIds: ReplicatorTest {
+    
+    #if COUCHBASE_ENTERPRISE
+    
```

Comment:
is this only an EE only feature? i can see its an CE feature

Reply:
It is CE feature but it needs to test under EE because it uses DatabaseEndpoint (EE only).